### PR TITLE
Jeremy/eng 1024 update help text + wifi fields

### PR DESCRIPTION
**Pull request by jerminatorhits**
## Description
Why did you write this code?
- Include minor edits, specifically to Amenities and adding WiFi
- Include Wifi Photo field to upload photos.
- refactor form css
- make shared bathroom text clickable

https://docs.google.com/document/d/1w1RCVl9lIHqGA5UOVk3B0zqgRG3qSDBlfsQP_iiF0hg/edit?ts=5c6d15f0

## How to Test
- [ ] Switch backend to https://github.com/thebeetoken/beenest-backend/pull/770
- [ ] Visit 'host/listings/:id/accommodations'
- [ ] add a WIFI photo using the photo uploader at the bottom and type in internet speed
- [ ] Click save and continue
- [ ] Navigate back to Accommodations to see uploaded photo populate the field and wifi speed 
- [ ] Refresh page and still see the same values
- [ ] click shared bathroom text and verify toggle
- [ ] Verify language and help text looks presentable

- [ ] Adjust browser size for regression.

Which devices did you test on?
- [ ] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/components/routes/Host/ListingForm/ListingHelp.tsx [hunk 1]
```diff
@@ -129,62 +130,32 @@ const SharedBathroom = () => (
...
 
 const Wifi = () => (
   <div>
-    <p>Include Wifi capabilities and working desk(s) for business travelers. Seperate listings with commas.</p>
     <p>Bonus points for</p>
     <ol>
-      <li>(1) going to https://fast.com/</li>
-      <li>(2) screenshotting the speed and adding it to your photos</li>
+      <li>Going to <a href="https://fast.com/">https://fast.com/</a></li>
+      <li>Screenshotting the speed and adding it to your photos</li>
```
Comment (tc) [inline src/components/routes/Host/ListingForm/ListingHelp.tsx]: did we want to change this not to add to photos but to a separate field?

Comment (jerminatorhits) [reply src/components/routes/Host/ListingForm/ListingHelp.tsx]: ya. making separate field just for wifiPhoto

File: src/components/routes/Host/ListingForm/AccommodationsForm/AccommodationsForm.tsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ import NumberInput from 'shared/NumberInput';
 import Textarea from 'shared/Textarea';
 import { TextareaEvent } from 'shared/Textarea/Textarea';
 import { stringToArray, arrayToString } from 'utils/formatter';
+import { PhotoUploader, Photo } from 'components/shared/PhotoUploader';
```
Comment (kcvan) [inline src/components/routes/Host/ListingForm/AccommodationsForm/AccommodationsForm.tsx]: `component` snuck in

Comment (jerminatorhits) [reply src/components/routes/Host/ListingForm/AccommodationsForm/AccommodationsForm.tsx]: removed 🙏

File: src/components/routes/Host/ListingForm/ListingForm.tsx [hunk 3]
```diff
@@ -56,6 +57,10 @@ const defaultValues: FormValues = {
   [ListingField.SLEEPING_ARRANGEMENT]: '',
   [ListingField.STATE]: '',
   [ListingField.TITLE]: '',
+  [ListingField.WIFI]: {
+    photo: '',
+    speed: 0,
```
Comment (tc) [inline src/components/routes/Host/ListingForm/ListingForm.tsx]: we should have the measurement unit as part of the name

Comment (jerminatorhits) [reply src/components/routes/Host/ListingForm/ListingForm.tsx]: changed `speed` to `mbps`

File: src/components/routes/Host/ListingForm/ListingHelp.tsx [hunk 4]
```diff
@@ -104,6 +106,7 @@ const SleepingArrangement = () => (
   <div>
     <h3>Sleeping Arrangement</h3>
     <p>Sleeping Arrangement must include the quantity and size of beds.</p>
+    <p>Don’t forget to add <span>#</span> and <span>type</span> of bed!</p>
```
Comment (woeltjen) [inline src/components/routes/Host/ListingForm/ListingHelp.tsx]: Noticed a few places where `span` is used for styling. Can this be something semantic, like `em`? Given that these `span`s are styled with `${typography('emp', 5)}` emphasis appears to be the intent.

Comment (jerminatorhits) [reply src/components/routes/Host/ListingForm/ListingHelp.tsx]: ty. changed to `<strong>` for bold since `<em>` is italics.
